core/src/test/scala/org/apache/spark/sql/delta/DeltaShowCreateTableSuite.scala

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta
+
+// scalastyle:off import.ordering.noEmptyLine
+
+import org.apache.spark.sql.{QueryTest, SparkSession}
+import org.apache.spark.sql.delta.test.DeltaSQLCommandTest
+import org.apache.spark.sql.test.SharedSparkSession
+import org.apache.spark.sql.types._
+
+import java.io.File
+import scala.reflect.io.Directory
+
+class DeltaShowCreateTableSuite extends QueryTest with SharedSparkSession with DeltaSQLCommandTest {
+
+  test(testName = "Test DDL Output for External Table SHOW CREATE TABLE") {
+    withTempDir { foo =>
+      withTable("external_table") {
+        val table = "external_table"
+        val fooPath = foo.getCanonicalPath
+        sql(s"CREATE TABLE `$table` (id LONG) USING delta LOCATION '$fooPath'")
+        val ddl = getShowCreateTable(table)
+        assert(ddl.contains("CREATE TABLE"))
+        assert(ddl.contains("'external' = 'true'"))
+        assert(ddl.contains(s"LOCATION 'file:$fooPath'"))
+        assert(ddl.contains("USING delta"))
+      }
+    }
+  }
+
+  test(testName = "Test DDL Output for Managed Table SHOW CREATE TABLE") {
```

Comment:
can you add a test case where you assert against the actual string? as opposed to just using `.contains`?